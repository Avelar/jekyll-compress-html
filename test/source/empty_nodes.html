---
layout: compress
---

<div><div>One</div><div> </div><div>Three</div><div></div></div>
