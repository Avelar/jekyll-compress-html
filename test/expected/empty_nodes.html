<div><div>One</div><div>Three</div></div>
